l10n_br_account_product/models/product.py

```diff
@@ -43,3 +62,8 @@ class ProductTemplate(models.Model):
 
     service_type_id = fields.Many2one(
         'l10n_br_account.service.type', u'Tipo de Serviço')
+
+    product_estimated_taxes_percent = fields.Float(
```

Comment:
@rvalyi @renatonlima i'm very busy next days, but in weekends and Pycon i will have time to review this PRs. Or i can use the last day sprint for it.